### PR TITLE
Ported old blog posts from neurostorm.io

### PR DESCRIPTION
**Pull request by loftusa**
Didn't change the dates, so they currently appear below "First Post", which is a bit awkward. I can change the dates to be today, but then those dates wouldn't coincide with when the posts were written. 

I'll redo this PR if the dates should be changed

### CODE REVIEW

**Comment by loftusa** [comment]
Changed dates, decided it was too weird to have the old blog posts appear underneath "first post"

File: _posts/2016-12-18-most-important-articles-for-us.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: Useful Background Reading for NeuroData
+description: ""
+modified: 2018-10-19
+category: misc
+author: jovo
```
Comment (falkben) [inline _posts/2016-12-18-most-important-articles-for-us.md]: Can you use his full name as in the other blog posts?

File: _posts/2017-01-17-what-is-statistics-for.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: What is statistics for
+description: ""
+modified: 2018-10-19
+category: misc
+author: jovo
```
Comment (falkben) [inline _posts/2017-01-17-what-is-statistics-for.md]: same here

**Review by falkben — CHANGES_REQUESTED** [review]
You have to change the file names to get the dates to change.  I don't know what the modified tag actually does.

**Comment by loftusa** [comment]
Should be good now

**Comment by loftusa** [comment]
Should be good now. Jovo wanted the old dates anyway.

**Pull request merged**